lesson_2/pkg/fib/fib.go

```diff
@@ -0,0 +1,25 @@
+package fib
+
+import "fmt"
+
+// NumL : count Fibonacci number by loop and print
+func NumL(n int) {
+	cur, prev :=  1, 0
+	for i := 0; i < n - 1; i++ {
+		prev, cur = cur, cur + prev
+	}
+	fmt.Println(cur)
+}
```

Comment:
function shouldn't print result. You should count and return result? to use it somewhere...